doc/ref_model/chapters/appendix-a.md

```diff
@@ -5,8 +5,9 @@
 ## Table of Contents
 * [A.1 Goals](#A.1)
 * [A.2 Intro and Terminology](#A.2)
-* [A.3 VNF Maturity Levels](#A.3)
+* [A.3 VNF Evolution Phases](#A.3)
```

Comment:
I prefer "Phases of Evolution" but not enough to hold things up.

Reply:
@nickchase Hi Nick, I was just making a small change to the language, but the actual implementation is being done under a different PR, so I have interjected you comment in said PR, at the following link: https://github.com/cntt-n/CNTT/pull/754#issuecomment-562574042.

If this action addresses your concern with regard to this PR, please mark this sub-thread as Resolved. Thanks, -Mark